kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/MarshallingFormat.java

```diff
@@ -42,14 +42,14 @@ public static MarshallingFormat fromId( int id ) {
     }
 
     public static MarshallingFormat fromType( String type ) {
-        if ("xstream".equalsIgnoreCase( type )) {
+        if ("xstream".equalsIgnoreCase( type ) || "application/xstream".equalsIgnoreCase( type ) ) {
             return XSTREAM;
-        } else if ("xml".equalsIgnoreCase( type )) {
+        } else if ("xml".equalsIgnoreCase( type ) || "application/xml".equalsIgnoreCase( type ) ) {
```

Comment:
now I know I didn't add that - again due to dependency - it's kie-server-api so it's not depending on and "rest" stuff so I don't want to add it here just to avoid too much being added